zerver/lib/events.py

```diff
@@ -157,6 +158,11 @@ def fetch_initial_state_data(user_profile, event_types, queue_id,
     if want('realm_bot'):
         state['realm_bots'] = get_owned_bot_dicts(user_profile)
 
+    # This does not yet have an apply_event counterpart, since currently,
+    # new entries for EMBEDDED_BOTS can only be added directly in the codebase.
+    if want('realm_embedded_bots'):
+        state['realm_embedded_bots'] = list(bot.name for bot in EMBEDDED_BOTS)
+
```

Comment:
In terms of things that are easy to merge now without adding the feature, this section `realm_embedded_bots` code could easily be submitted as its own PR and we could just merge it.  Feel free to do that if you like.

(Just thinking about what pieces we can merge without turning the feature on)

Reply:
I split out a respective commit and moved it to the beginning of the PR, so you can easily merge it. Out of interest, does it make any difference for you if the commit is in the beginning or in the middle?

Reply:
Great, merging that preliminary as soon as the tests pass.  

It doesn't have to be at the beginning, but it save a bit of time when rebasing, and it's also a way to communicate which things can be merged first :)